app/views/account_mailer/account_activation.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= t'.title' %></h1>
+
+<p><%= t'.welcome_html', user: @user.name %></p>
+
+<p>
+Welcome to VirtualTable! Click on the link below to activate your account:
+</p>
+
```

Comment:
You can remove the `<p>` and `</p>` here. We can just use HTML in the locale. You can also remove the "Welcome to VirtualTable! Click on the link below to activate your account:" line because we will also add that in the translation.